src/PFXImportPowershell/README.md

```diff
@@ -5,18 +5,30 @@ This project consists of helper Powershell Commandlets for importing PFX certifi
 These scripts provide a baseline for the actions that can take place to import your PFX Certificates to Intune. They can be modified and adapted to fit your workflow. Most of the cmdlets are wrappers of Intune Graph calls.
 
 ## What's New?
+
+### Version 1.2
+- Breaking changes:
+	- The global Intune app registration is deprecated and its client ID has been removed from these scripts.  A tenant-specific app registration must be created and its client ID added to your IntunePfxImport.psd1 file.
+	- The previously deprecated Get-IntuneAuthenticationToken command has been removed.  Use Set-IntuneAuthenticationToken instead.  The associated AuthenticationResult parameter has also been removed from the other various commands.
+	- Changed the default redirect uri to https://login.microsoftonline.com/common/oauth2/nativeclient as recommended by Microsoft Azure.  
```

Comment:
Will this affect the customer at all?  Would they notice it? If not, we might want to remove it.

Reply:
it affects any that had previously created an app registration with the old redirect uri. (they can then update their app registration or override the default in the .psd1 file)